backend/controllers/movie.mjs

```diff
@@ -9,12 +9,12 @@ const movieCtrl = Router()
 movieCtrl.get('/', authUser , async (req,res,next) => {
     try{
         const userId = req.body.user._id;
-        const movies = await getUserAsync(userId)
-        .then(user => getMoviesAsync(user.watchList) ).catch(() => null);
```

Comment:
Try to use only async await without then
Also inside async function you don't need to use then

Reply:
Noted! tnx Achiya